src/user-guide/writing-workflows/runtime.rst

```diff
@@ -411,49 +424,68 @@ Dynamic Platform Selection
-explained in :ref:`WhitherStdoutAndStderr`. For remote tasks
-the same directory is used, but *on the task host*.
-Remote task log directories, like local ones, are created on the fly, if
-necessary, during job submission.
+Task job stdout and stderr streams are written to :term:`log files <job log>`
+under the workflow :term:`run directory` (see :ref:`WhitherStdoutAndStderr`).
+For remote tasks the same directory is used, on the job host.
 
 
 .. _ImplicitTasks:
 
 Implicit Tasks
 --------------
 
-An :term:`implicit task` appears in the workflow graph but has no
-explicit runtime configuration section. Such tasks automatically
-inherit the configuration from the root namespace.
-This is very useful because it allows functional workflows to
-be mocked up quickly for test and demonstration purposes by simply
-defining the graph. It is somewhat dangerous, however, because there
-is no way to distinguish an intentional implicit task from one
-caused by typographic error. Misspelling a task name in the graph
-results in a new implicit task replacing the intended task in the
-affected trigger expression, and misspelling a task name in a runtime
-section heading results in the intended task becoming an implicit task
-itself (by divorcing it from its intended runtime config section).
-
-You can allow implicit tasks during development of a workflow using
-:cylc:conf:`flow.cylc[scheduler]allow implicit tasks`. But, to avoid
-the problems mentioned above, any task used in a production/operational
-workflow should not be implicit, i.e. it should have an explicit entry in under
-the runtime section of ``flow.cylc``, even if the section is empty. This
-results in exactly the same task behaviour, via inheritance from root,
-but adds a layer of protection against mistakes. Thus, it is recommended to
-turn off :cylc:conf:`flow.cylc[scheduler]allow implicit tasks` when the
-:cylc:conf:`flow.cylc[runtime]` section has been written.
+An implicit task is one that appears in the graph but is not defined under
+:cylc:conf:`flow.cylc[runtime]`.
+
+Depending on the value of :cylc:conf:`flow.cylc[scheduler]allow implicit tasks`
+Cylc can automatically create default task definitions for these, to submit
+local dummy jobs that just return the standard job status messages.
+
+Implicit tasks can be used to mock up functional workflows very quickly. A
+default ``script`` can be added to the root family, e.g. to slow job execution
+down a little. Here is a complete workflow definition using implicit tasks:
+
+.. code-block:: cylc
+
+   [scheduler]
+       allow implicit tasks = True
+   [scheduling]
+       [[graph]]
+           R1 = "prep => run-a & run-b => done"
+   [runtime]
+       [[root]]
+           script = "sleep 10"
+
+
+.. warning::
+  Implicit tasks are somewhat dangerous because they can easily be created by
+  mistake: misspelling a task's name divorces it from its ``runtime`` definition.
```

Comment:
```suggestion
   Implicit tasks are somewhat dangerous because they can easily be created by
   mistake; misspelling a task's name divorces it from its ``runtime`` definition.
```

Reply:
Applied the indent, but I think the colon is correct here. The second clause explains how implicit tasks can be created by mistake.